pkg/sdk/resource_monitors.go

```diff
@@ -97,7 +98,13 @@ func (row *resourceMonitorRow) convert() (*ResourceMonitor, error) {
 		resourceMonitor.Frequency = *frequency
 	}
 	if row.StartTime.Valid {
-		resourceMonitor.StartTime = row.StartTime.String
+		const YYMMDDhhmm = "2006-01-02 15:04"
```

Comment:
There is a test `TestInt_ResourceMonitorAlter`#`when changing scheduling info` in `resource_monitors_integration_test.go`. It uses `ParseTimestampWithOffset` that already uses similar logic, but it's run only for test check, and not the implementation.

Maybe this should be reused/merged with your proposed solution?

Reply:
I'll take a look - thanks

Reply:
Hey, I've moved parsers.go into sdk and updated with a merged parse timestamp function. Updated the integration tests and used the function in the resource monitor update.

Reply:
Hey, did you revert it? I can't see the move of `parsers.go` currently.

Reply:
Hey -my branch reverted (didn't do this intentionally), I've re-pushed all the changes, hopefully at parity now!